packages/x-components/src/x-modules/empathize/store/types.ts

```diff
@@ -40,6 +40,12 @@ export interface EmpathizeMutations {
    * @param isOpen - The new isOpen state to save.
    */
   setIsOpen(isOpen: boolean): void;
+  /**
+   * Sets the {@link EmpathizeState.config } config.
+   *
+   * @param config - The new config.
+   */
+  setConfig(config: EmpathizeConfig): void;
```

Comment:
This type already exists in the `config-store.utils.ts`. You have to import the mutations type and make the mutations extend that type.

Reply:
Hi, regarding this, if we extend the mutations we shouldn't need to declare them here, so we could remove `setConfig` & `mergeConfig` types from here. If this is okay, why are we keeping `setQuery` mutation types in modules if we are extending `QueryMutations`??